static/manifest.json

```diff
@@ -1,27 +1,25 @@
 {
-    "name": "Otto Tabs",
-    "description" : "Smart tabs management",
-    "version": "0.2.2",
-    "manifest_version": 2,
-    "permissions": ["tabs", "storage"],
-    "browser_action": {
-      "default_popup": "index.html"
-    },
-    "background": {
-      "page": "background.html",
-      "persistent": false
-    },
-    "commands": {    },
-    "icons": { 
-      "48": "ic_launcher_round_48.png",
-      "72": "ic_launcher_round_72.png",
-      "96": "ic_launcher_round_96.png",
-      "144": "ic_launcher_round_144.png",
-      "192": "ic_launcher_round_192.png"
-    }
-  }
-
-  
-
-
-
+	"name": "Otto Tabs",
+	"description": "Smart tabs management",
+	"version": "0.2.2",
+	"manifest_version": 3,
```

Comment:
"minimum_chrome_version": "88" could be added to prevent [older versions that do not support Manifest V3](https://developer.chrome.com/docs/extensions/migrating/)

Reply:
Resolved starting in: fb7ae9b4968a0e02c15e01e53f39dc9b7d7f8cc5